datagrepper/app.py

```diff
@@ -381,11 +350,11 @@ def raw():
 
         # Convert our messages from sqlalchemy objects to json-like dicts
         messages = [msg.__json__() for msg in messages]
-        if grouped:
+        if grouped:  # pragma: no cover
```

Comment:
We'll remove the "grouped" feature because fedora-messaging does not support it, but it should be the focus of a different PR. In the meantime, we shouldn't exclude it from coverage.